### ISSUE TITLE
Upgrade to rust stable

### ISSUE THREAD

**Issue by ulrichard** [dr-orlovsky/rust-glade#1]
I'm trying to compile bitcoin-pro, which depends on rust-glade.
On debian testing, installing a couple of packages got me across the first compiler errors. But it still fails on glade.
- libglib2.0-dev 
- libcairo2-dev
- libgdk-pixbuf-2.0-dev
- libatk1.0-dev
- libpango1.0-dev
- libgtk-3-dev

**on debian testing with rustc 1.48.0:**
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> /home/richi/.cargo/registry/src/github.com-1ecc6299db9ec823/glade-0.1.0-alpha.4/src/lib.rs:15:1
   |
15 | #![feature(try_trait)]
   | ^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0554`.
error: could not compile `glade`.

**On ubuntu 21.04 with rustc 1.55:**
 --> /home/richi/.cargo/registry/src/github.com-1ecc6299db9ec823/glade-0.1.0-alpha.4/src/error.rs:22:25
std::option::NoneError not found in std::option 

When I try to compile rust-glade master on the ubuntu machine, I get the same error as listed above under debian testing.

**Comment by ulrichard** [comment]
Yeah, found the culprit: 
https://doc.rust-lang.org/std/option/struct.NoneError.html
Any idea how long it will take for this to be available in the stable compiler? Looks like the issue has been open for four years:
https://github.com/rust-lang/rust/issues/42327

**Comment by ulrichard** [comment]
I run into the same problem when I try with docker: https://github.com/pandoracore/bitcoin-pro/blob/master/contrib/docker-builder/README.md

**Comment by dr-orlovsky** [comment]
Yeah, because of `NoneError` this library is nightly-only. Rust stabilization of this feature has being stucked in endless debates on a proper way of doing `Try` trait, and this year they refactored everything once again, and I assume it will take till about end of the year to get it into stable (I assume they will add it to the 2021 edition).

Nevertheless I am targeting to remove this feature dependency, so in my next round when I will be working on bitcoin pro I will fix this. For now, just compile after running `rustup default nightly` and that's all
